server/service/service_activities.go

```diff
@@ -0,0 +1,13 @@
+package service
+
+import (
+	"context"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+)
+
+func (svc *Service) ListActivities(ctx context.Context, opt fleet.ListOptions) ([]*fleet.Activity, error) {
+	if err := svc.authz.Authorize(ctx, &fleet.Invite{}, fleet.ActionRead); err != nil {
```

Comment:
```suggestion
	if err := svc.authz.Authorize(ctx, &fleet.Activity{}, fleet.ActionRead); err != nil {
```

We will then need to add the appropriate policy to the policy file.

Reply:
Whoops, copy paste error.